shiny/session/_session.py

```diff
@@ -608,22 +608,26 @@ def _send_remove_ui(self, selector: str, multiple: bool) -> None:
     @overload
     def _send_progress(
         self, type: Literal["binding"], message: BindingProgressMessage
-    ) -> None: ...
```

Comment:
These were to resolve a flake8 error, #704

Reply:
IDK about the GitHub issue link... but the code change is good!

Reply:
Just FYI, after #1125, it's not necessary to make these changes. That PR updated the flake8 configuration as recommended by Black, which allows this formatting.

https://github.com/posit-dev/py-shiny/pull/1125/files#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52